src/core/src/core_logic/PatchInstaller.py

```diff
@@ -186,30 +185,30 @@ def install_updates(self, maintenance_window, package_manager, simulate=False):
 
         stopwatch_for_batch_install_process.stop()
 
-        # Check for packages available to install. After installing packages in batches, there might be still some packages available for installation. These packages should be:
-        # (a) Due to not enough remaining time in maintenance window to install the packages in batch, installing packages in batch is stopped and hence some packages are not installed in batches
-        # (b) Those which were failed while installing in batches
-        # (c) Some new packages available in this brief time when installation in batches was being done
+        # Check for packages available and install them in batches. Some packages may not be installed due to:
+        # (a) Not enough time remaining in maintenance window
+        # (b) Failure during package installation
+        # (c) New packages added during the brief time batch installation was in progress
         packages, package_versions = self.get_remaining_packages_to_install(package_manager)
 
-        batch_processing_perf_log  = {"Installed packages count in batch processing": str(installed_update_count), "number of remaining packages to install": str(len(packages)), 
-                                      Constants.PerfLogTrackerParams.PATCH_OPERATION_SUCCESSFUL: str(patch_installation_successful), "Stopped installation in batch due to not enough remaining time to install in batches": str(maintenance_window_batch_cutoff_reached)}
+        batch_processing_perf_log  = {Constants.PerfLogTrackerParams.TASK: "Installation of packages in batches", "Installed packages count in batch processing": str(installed_update_count), "number of remaining packages to install": str(len(packages)), 
```

Comment:
nit:Can this be a Unique string like "BatchProcessingInstalledPackagesCount".  It may help in improving your queries later.,

Reply:
Same for other places for log collection

Reply:
Is this still pending? @GAURAVRAMRAKHYANI

Reply:
This is done now. I had kept it pending as there was a separate PR for log structure changes https://github.com/Azure/LinuxPatchExtension/pull/184

I wanted to wait till that log structure change PR is completed so that I could follow same log structure in this PR.